### DISCUSSION TITLE
Content-relative Anchors

### DISCUSSION THREAD

**Discussion by timfi** [cetz-package/cetz#680] [Q&A] [answered]
I'm currently playing around with Cetz and am finding myself missing the more obscure anchors that Tikz provides. In particular, I'm missing the the anchors related to the properties of a nodes content—see `base*`, `mid*`, and `text` in the image below. After some adventures through both the manual and the source code afterwards I've come to the sad conclusion that Cetz does not support these (yet?); not even for `content` nodes. Is this conclusion correct? And if so is this worth an issue or is it out of scope for Cetz?

![grafik](https://github.com/user-attachments/assets/4ae761b1-7bc8-48d2-9626-0dce10fd4c66)
(Credits to [Packt](https://web.archive.org/web/20240813122622/https://subscription.packtpub.com/book/business-and-other/9781804618233/3/ch03lvl1sec22/using-shapes-and-anchors) for the image.)

**Comment by fenjalien** [comment, marked as answer]
No we currently don't as it would require inspecting the provided content and measuring it. This is difficult as theres no official documentation for how content values are supposed to act and could change with any release. Once Typst provides more ways to control content it would be something we'll be looking into.

Kind of related to #395

**Reply by timfi** [reply]
Ok, that makes sense. Should I create seperate issue to keep track of this or should I just tack it onto #395?

**Reply by fenjalien** [reply]
A seperate issue would be better :)

**Reply by timfi** [reply]
Done (see #683) and thank you for the quick and helpful replies. 🙏
